assets/src/adaptivity/rules-engine.ts

```diff
@@ -123,9 +123,8 @@ const processRules = (rules: JanusRuleProperties[], env: Environment) => {
             actualValue = ogValue;
           }
         } else if (ogValue.lastIndexOf(',') !== -1) {
-          const tolerance = ogValue.split(',');
-          actualValue = tolerance[0];
-          toleranceValue = tolerance.length == 2 ? tolerance[1] : 0;
+          toleranceValue = ogValue.substring(ogValue.lastIndexOf(',') + 1);
```

Comment:
There was an issue when separating the tolerance value and the actual value. Due to that the wrong expression was getting calculated. This fixes that